triton/kubernetes/AWS/FIL/README.md

```diff
@@ -0,0 +1,215 @@
+## Deploying a Triton service that uses a custom plugin for the RAPIDS cuML Forest Inference Library (FIL).
+
+**NOTE:** For steps to setup a horizontally autoscalable Triton Service in EKS refer to the instructions in [Detailed_HPA_Setup](./Detailed_HPA_Setup.md).
+
+### Overview
+
+This example will illustrate the workflow to deploy a Triton inference service, with a [non-trival custom backend](https://github.com/triton-inference-server/fil_backend) for accelerated forest model inference on [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc&eks-blogs.sort-by=item.additionalFields.createdDate&eks-blogs.sort-order=desc). 
```

Comment:
Could we change the wording on `non-trivial custom backend`? We're trying to make it clear generally that FIL is *not* a custom backend but a fully integrated and supported part of Triton.